src/electron.d.ts

```diff
@@ -10052,6 +14058,14 @@ declare namespace Electron {
     height: number;
   }
 
+  interface WinEventHookEmitterConstructorOptions {
```

Comment:
This is changed per [this runtime PR](https://github.com/openfin/runtime/pull/1490/files#diff-d8276236bae98c946ba4bf143f66d0f6R66)